### PR TITLE
Add compat data for mtable (MathML)

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mtable

### CODE REVIEW

File: mathml/elements/mtable.json [hunk 1]
```diff
@@ -0,0 +1,1062 @@
+{
+  "mathml": {
+    "elements": {
+      "math": {
```
Comment (Elchi3) [inline mathml/elements/mtable.json]: math -> mtable

Comment (ExE-Boss) [inline mathml/elements/mtable.json]: This is `mtable`.

Comment (ExE-Boss) [reply mathml/elements/mtable.json]: I literally [just said that](#discussion_r188647208).

File: mathml/elements/mtable.json [hunk 2]
```diff
@@ -0,0 +1,1062 @@
...
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "align": {
+          "__compat": {
+            "description": "<code>align</code>",
```
Comment (Elchi3) [inline mathml/elements/mtable.json]: These descriptions aren't needed (remove throughout the file).

**Pull request merged**